activities.el

```diff
@@ -279,6 +279,11 @@ prefixes will be added automatically."
                  (const :tag "Current project's name" activities--project-name)
                  (function-item :tag "Other function")))
 
+(defcustom activities-inhibit-frame-name nil
```

Comment:
Naming an option in negative terms creates a "double-negative problem," where the user has to think in terms of negating a negative to understand what it does.  If we name it simply `-set-frame-name`, it will be clearer.